examples/shaders/shaders_basic_pbr.c

```diff
@@ -105,11 +106,11 @@ int main()
     shader.locs[SHADER_LOC_MAP_NORMAL] = GetShaderLocation(shader, "normalMap");
     // WARNING: Similar to the MRA map, the emissive map packs different information 
     // into a single texture: it stores height and emission data
-    // It is binded to SHADER_LOC_MAP_EMISSION location an properly processed on shader
+    // It is binded to the SHADER_LOC_MAP_EMISSION location and properly processed on the shader
```

Comment:
I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.

Reply:
Not using "the" is fine.  However "binded" is not a word :).  I believe the correct term is "bound."

Reply:
> Not using "the" is fine. However "binded" is not a word :). I believe the correct term is "bound."

Shoot you are right! my bad